tests/test_metamod.py

```diff
@@ -25,3 +31,37 @@ def test_metaswap_dll_dep_dir_contains_dependencies(metaswap_dll_dep_dir: Path)
 
 def test_modflow_dll_present(modflow_dll: Path) -> None:
     assert modflow_dll.is_file()
+
+
+def test_metamod_sprinkling(
+    tmp_path: Path,
+    coupled_mf6_model: Modflow6Simulation,
+    msw_model: MetaSwapModel,
+    metaswap_lookup_table: Path,
+    metaswap_dll: Path,
+    metaswap_dll_dep_dir: Path,
+    modflow_dll: Path,
+):
+    # Override unsat_svat_path with path from .env
+    msw_model.simulation_settings[
+        "unsa_svat_path"
+    ] = msw_model._render_unsaturated_database_path(metaswap_lookup_table)
+
+    metamod = MetaMod(
+        msw_model,
+        coupled_mf6_model,
+        mf6_rch_pkgkey="rch_msw",
+        mf6_wel_pkgkey="wells_msw",
+    )
+
+    metamod.write(
+        tmp_path,
+        modflow6_dll=modflow_dll,
+        metaswap_dll=metaswap_dll,
+        metaswap_dll_dependency=metaswap_dll_dep_dir,
+    )
+
+    os.chdir(tmp_path)
```

Comment:
Chdir is not necessary, you can just specify the full path of the toml

Reply:
Removed the chdir